ci/build_docs.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash
-# Copyright (c) 2020-2024, NVIDIA CORPORATION.
+# Copyright (c) 2020-2025, NVIDIA CORPORATION.
 
 set -euo pipefail
 
 rapids-logger "Create test conda environment"
+
+# shellcheck source=/dev/null
 . /opt/conda/etc/profile.d/conda.sh
 
 RAPIDS_VERSION="$(rapids-version)"
-export RAPIDS_VERSION_MAJOR_MINOR="$(rapids-version-major-minor)"
+RAPIDS_VERSION_MAJOR_MINOR="$(rapids-version-major-minor)"
+export RAPIDS_VERSION_MAJOR_MINOR
```

Comment:
I do not like this style because it requires two lines where only one was used before. `export VAR="value"` seems like a very common and well-understood pattern in shell scripts. Is there a justification for why this is better practice?

Reply:
doing it as a single line masks the return value of the captured command.  so if the captured command fails, even with `set -e`, the script won't fail because the _export_ still succeeds.

Reply:
TIL! I need to read the shellcheck docs, probably.

Reply:
`bash` is a gnarly language.  footguns all the way down